src/main/java/Timo/Timo.java

```diff
@@ -0,0 +1,584 @@
+    }
+}
+
+class Todo extends Task {
+    private String tasktype = "T";
+
+
+    public Todo(boolean mark, String val) {
+        super(mark, val);
+    }
+
+
+    @Override
+    public String toString() {
+        return "[" + this.tasktype + "]" + super.toString();
+    }
+}
+
+class Deadline extends Task {
+    private LocalDateTime date;
+    private String tasktype = "D";
+
+    public Deadline(boolean mark, String val, LocalDateTime date){
+        super(mark, val);
+        this.date = date;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + this.tasktype + "]" + super.toString() +
+                " (by: " + this.date.format(DateTimeFormatter.ofPattern("MMM dd yyyy HHmm"))
+                + ")";
+    }
+}
+
+class Event extends Task {
+    private String from;
+    private String to;
+    private String tasktype = "E";
+
+    public Event(boolean mark, String val, String from, String to) {
+        super(mark, val);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + this.tasktype + "]" + super.toString() + " (from: " + this.from + " to: " + this.to + ")";
+    }
+}
+
+class TimoException extends Exception {
+
+    public TimoException(String errorMessage) {
+        super(errorMessage);
+    }
+
+}
+
+//Timo.Storage: beginning I read from the file, end I update the file
+class Storage {
+    private final String filepath;
+
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Loads tasks from a file specified by the `filepath` and returns them as a list of `Task` objects.
+     * <p>
+     * The tasks in the file are expected to be in a specific format, indicating the type of task
+     * (Todo, Deadline, or Event) and its completion status.
+     * </p>
+     *
+     * @return A `List` of `Task` objects representing the tasks stored in the file.
+     *
+     * @throws FileNotFoundException if the file at the specified `filepath` does not exist.
+     *
+     * @see Task
+     * @see Todo
+     * @see Deadline
+     * @see Event
+     */
+    public List<Task> load() throws FileNotFoundException {
+        File f = new File(this.filepath);
+
+        //initialise array to store the values
+        List<Task> arr = new ArrayList<Task>();
+
+        //check if the file exists
+        if (f.exists()) {
+            Scanner s = new Scanner(f);
+            while (s.hasNext()) {
+                String tmp = s.nextLine();
+                if (tmp.startsWith("[T]")) {
+                    String[] a = tmp.split("] ", 2);
+                    if (Character.compare(tmp.charAt(4), 'X') == 0) {
+                        arr.add(new Todo(true, a[1]));
+                    } else {
+                        arr.add(new Todo(false, a[1]));
+                    }
+                } else if (tmp.startsWith("[D]")) {
+                    //remove the [D][?] from the line
```

Comment:
I like how comments are added to facilitate understanding how the code creates a deadline and event, perhaps similar comments could be added into todo as well?

Reply:
Ok I will do that, thanks!